src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -104,7 +132,10 @@ public string GetConfigFileName()
         {
             Console.WriteLine($"Loading config file from {path}.");
             string json = _fileSystem.File.ReadAllText(path);
-            return TryParseConfig(json, out config, connectionString: _connectionString, replaceEnvVar: replaceEnvVar);
+            bool res = TryParseConfig(json, out RuntimeConfig, connectionString: _connectionString, replaceEnvVar: replaceEnvVar);
+            TrySetupConfigFileWatcher();
+            config = RuntimeConfig;
+            return res;
         }
         else
         {
```

Comment:
The code inside else block can be moved out of it and the else block can be removed.